game_eggs/steamcmd_servers/stationeers/egg-stationeers.json

```diff
@@ -4,18 +4,18 @@
         "version": "PTDL_v1",
         "update_url": null
     },
-    "exported_at": "2022-01-20T13:00:48-05:00",
+    "exported_at": "2022-06-18T13:00:00-00:00",
     "name": "Stationeers",
     "author": "sysadmin@whiteshield.ch",
-    "description": "Stationeers Server\r\n\r\ndefault.ini will be created once you start game. Command parameters override default.ini parameters\r\nDon't forget to change the RCON Password !!\r\n\r\nRemote Administrator:\r\nYou can send commands on web browser.\r\nLink : http:\/\/[dedicated server address]:[GamePort]\r\n\r\nDedicated Server Wiki ==> https:\/\/stationeers-wiki.com\/Dedicated_Server_Guide\r\n\r\nStopping the server don't save the game, for manual save use the Remote Administrator.",
+    "description": "Stationeers Server\r\n\r\nsetting.xml will be created once you start game. Command parameters override setting.xml parameters\r\n\r\nDedicated Server Wiki ==> https:\/\/stationeers-wiki.com\/Dedicated_Server_Guide\r\n\r\nStopping the server don't save the game, for manual save use the Console.",
     "features": [
         "steam_disk_space"
     ],
     "images": [
         "quay.io\/pterodactyl\/core:source"
     ],
     "file_denylist": [],
-    "startup": ".\/rocketstation_DedicatedServer.x86_64 -batchmode -nographics -autostart -autosaveinterval={{SAVE_INTERVAL}} -clearallinterval={{CLEAR_INTERVAL}} -worldtype={{SERVER_MAP}} -worldname={{SAVE_NAME}} -loadworld={{SAVE_NAME}} -basedirectory=\/home\/container -updateport={{UPDATE_PORT}} -gameport={{GAME_PORT}}",
+    "startup": ".\/rocketstation_DedicatedServer.x86_64 -loadlatest {{SAVE_NAME}} {{SERVER_MAP}} -settings ServerVisible true ServerName {{SERVER_NAME}} AutoSave true SaveInterval {{SAVE_INTERVAL}} UpdatePort {{UPDATE_PORT}} GamePort {{GAME_PORT}}",
```

Comment:
This startup will ends up in an error. In Docker, ther server must be started with -it flag. else server will not work. Ptero pick consoleoutput as i know via API